app/models/itinerary.rb

```diff
@@ -0,0 +1,4 @@
+class Itinerary < ApplicationRecord
+  belongs_to :trip, null: false
+  belongs_to :planet, null: false
```

Comment:
`null: false` is used when adding a constraint to a column on a table during a migration, but we dont need to use it on the model level.